src/test/kotlin/uk/gov/justice/digital/hmpps/prisonregister/resource/PrisonResourceIntTest.kt

```diff
@@ -59,6 +60,35 @@ class PrisonResourceIntTest : IntegrationTest() {
     }
   }
 
+  @Suppress("ClassName")
+  @Nested
+  inner class getPrisonFromActiveAndTextSearch {
+    @Test
```

Comment:
Would be useful to have a test to ensure default values are picked up correctly and don't cause the call to your endpoint to fall over

Reply:
Good point. Added the test `no search params provided` which does not pass any `queryParam`s